src/main/java/io/candydoc/ddd/extract_domain_vocabulary/ReflectionsDomainVocabularyFinder.java

```diff
@@ -161,10 +198,27 @@ private static Set<CanonicalName> dependenciesOf(
     return dependenciesOfTacticalPattern(domainModel);
   }
 
+  private static List<Class<?>> getFieldType(Field field) {
+    Type fieldType = field.getGenericType();
+    if (fieldType instanceof ParameterizedType) {
+      List<Class<?>> actualArgumentClasses = new ArrayList<>();
+      Type[] actualTypeArguments = ((ParameterizedType) fieldType).getActualTypeArguments();
+      for (Type type : actualTypeArguments) {
+        if (type instanceof Class) {
+          actualArgumentClasses.add((Class<?>) type);
+        }
+      }
+      return actualArgumentClasses;
+    }
+    return List.of(field.getType());
```

Comment:
Not sure to understand what this method does

Reply:
Before, the plugin didn't recognize the dependency if the dependency were the generic type of a list (like List<MyValueObject>). This function resolve that issue.